database/src/factories/login-user-backup.factory.ts

```diff
@@ -0,0 +1,17 @@
+import Faker from 'faker'
+import { define } from 'typeorm-seeding'
+import { LoginUserBackup } from '../../entity/LoginUserBackup'
+import { LoginUserBackupContext } from '../interface/UserContext'
+
+define(LoginUserBackup, (faker: typeof Faker, context?: LoginUserBackupContext) => {
+  if (!context) context = {}
+  if (!context.userId) throw new Error('LoginUserBackup: No userId present!')
```

Comment:
The logic is slightly different - this will also fail - the result is the same as far as I can see.
```suggestion
  if (!context || !context.userId) {
    throw new Error('LoginUserBackup: No userId present!')
  }
```